### ISSUE TITLE
AI Challenger trained HRNet weights

### ISSUE THREAD

**Issue by hongsukchoi** [leoxiaobin/deep-high-resolution-net.pytorch#207]
Hi, thanks for your great work. 

At `README.md`, there's an evaluation result of HRNet trained on MSCOCO+__AI Challenger__. (Indicated with *)
Where can I found the pretrained weights  of that model? I can't find it on the links you listed.
Thanks in advance.
<img width="1071" alt="스크린샷 2020-09-09 17 51 27" src="https://user-images.githubusercontent.com/31090011/92577197-5c586f00-f2c5-11ea-8ab0-da8a6ac2a176.png">

**Comment by dzyjjpy** [comment]
@hongsukchoi have you download AI Challenge datasets?  I cann't access to it's website

**Comment by hongsukchoi** [comment]
@dzyjjpy No, I can't either. I wonder why they don't provide the best model weight and accessible train dataset.

**Comment by dzyjjpy** [comment]
@hongsukchoi one friend share me the AI challenge datasets, FYI:
https://pan.baidu.com/s/1YFzoIkv51Yy_WjCvGv77qw#list/path=%2F

**Comment by hongsukchoi** [comment]
@dzyjjpy Thanks for your kindness. But I am having trouble downloading it. Do I have to sign in to Baidu?
